setup.py

```diff
@@ -41,11 +41,12 @@ def find_version(*file_paths):
     test_suite = 'tests',
 
     author = 'Iskandarov Eduard',
-    author_email = 'e.iskandarov@corp.mail.ru',
+    author_email = None,
```

Comment:
in case you retain author_email parameter - you could use my personal email: `eduard.iskandarov@ya.ru`